plugins/setup.py

```diff
@@ -50,6 +50,9 @@
             'package_langpacks=pulp_rpm.plugins.db.models:PackageLangpacks',
             'yum_repo_metadata_file=pulp_rpm.plugins.db.models:YumMetadataFile',
             'iso=pulp_rpm.plugins.db.models:ISO'
+        ],
+        'pulp.auxiliary_models': [
+            'erratum_pkglists=pulp_rpm.plugins.db.models:ErratumPkglist'
```

Comment:
It is loaded here pulp/pulp#2999.
Any other ideas to load this model and make sure indexes are there?